source/date_time.md

```diff
@@ -0,0 +1,89 @@
+# Date and time question types
+
+There are 3 different date and time question types in KoboToolbox: “Date”,
+“Time” and “Date and time”.
+
+The “Date” question type is for capturing date values, for example on a question
+like “What is your date of birth?” In both KoboCollect and Enketo web forms, a
+calendar-style date picker will be shown for selecting the date.
+
+The “Time” question type is for capturing time values, for example on a question
+like “At what time do you leave for work?” In both KoboCollect and Enketo, a
+time picker is shown where a user can select their response.
+
+The third type “Date and time” is for capturing both date and time responses on
+a single question.
+
+## How to set up Date and Time question types
+
+### Setting up in FormBuilder
+
+Adding “Date”, “Time”, and “Date and Time” questions is simple and does not
+require any extra setup:
```

Comment:
I think saying "and does not require any extra step" is unnecessary — there are very few instances when extra steps such as uploading media is required for questions. It might also be misleading as you do include the somewhat extra step of adding appearances lower down. Unless you have something else in mind when thinking of what extra steps there might be?

Reply:
I have edited the statement